diff --git a/wsgi.py b/wsgi.py
index d66f9c5..86615de 100644
--- a/wsgi.py
+++ b/wsgi.py
@@ -1,6 +1,8 @@
+#!/usr/bin/python
+
 import runner_service.configuration as configuration
 from runner_service.app import create_app
-from ansible_runner_service import setup_common_environment, remove_artifacts_init
+from runner_service.ansible_runner_service import setup_common_environment, remove_artifacts_init


 """
